packages/server/computation_container/random/csprng.cpp

```diff
@@ -13,7 +13,7 @@ bool CSPRNG::entropyCheck()
 {
     while (sodium_init() < 0)
     {
-        static unsigned int ctr = 0;
+        static uint16_t ctr = 0;
```

Comment:
```suggestion
        static std::uint16_t ctr = 0;
```

Reply:
Is the definition of `uint16_t` coming from the C header?

Reply:
[0dd20ee](https://github.com/acompany-develop/QuickMPC/pull/156/commits/0dd20ee8ce52c40ff109a40513a982689cf343cd)